### PR TITLE
Revert "Human nightvision"

### PR DESCRIPTION
**Pull request by optimumtact**
Reverts tgstation/tgstation#21638

This is causing a number of people to have fully white/black screens, one such report was windows 7/intel
@MrPerson @tgstation/commit-access thoughts?

### CODE REVIEW

**Comment by ChangelingRain** [comment]
i will bet you it's \~planes\~

**Comment by optimumtact** [comment]
I BET YOU IT'S BYOND

**Comment by ChangelingRain** [comment]
planes are byond!

**Comment by kevinz000** [comment]
you're a plane?

**Comment by AnturK** [comment]
How prevalent this is ?

**Comment by Shapsy** [comment]
Oh is that why I had white screens last night? Neat

**Comment by FantasticFwoosh** [comment]
Im a little confused how a publicly disliked by over ten people and 33 discussion points where it got heatedly debated between (this is bad - this is why + literally why) got put through anyway.

Just a thought, was the OP testing with their computer gamma/light wrenched up way high? Because of course if they adjust the lighting to be darker on their rig, they need to have some kind of mutual hardware test compared to all the mary sue's at home who play on different light settings.

**TL;DR** - They made it too dark as per request because they were playing too high is my opinion, no planes or engine issues.

Glad to see its on its way out now hopefully anyway.

**Comment by optimumtact** [comment]
Because it was changed and I liked it.

People don't need to be in complete pitch blackness in the dark

**Comment by optimumtact** [comment]
@AnturK it is, and it was roughly 5-6 people iirc

**Comment by MrPerson** [comment]
Commentary: The "white boxes/black boxes" are the lighting plane being drawn directly without applying the effects of the plane master. Also they're not even being added correctly it seems, which is double bizarre.

Anyways this is a BYOND bug that's probably not fixed in the current beta.  While I would still recommend people upgrade to it since it might help and it's better anyway, a revert is probably correct.

I would like to gather as much information from people who have this issue though, especially their GPU and BYOND version.

**Comment by optimumtact** [comment]
Damn I really wanted this stuff in ahhhhhhhhg

**Comment by MrPerson** [comment]
How many people is this affecting? A lot? A few? Because if you're willing to eat that loss, ok fine, but personally I'm kinda unhappy making people unable to play for this, at least in the short term.

Also obviously this should get reported at some point so lummox can fix it. Shocked the goons didn't run into this tbh, maybe they know something.

**Comment by MrStonedOne** [comment]
Its byond failing hardware rendering, Most are laptops with hybrid graphics card, I have a bug report up: http://www.byond.com/forum/?post=2176950 and there are instructions on a client workaround here:

http://www.byond.com/forum/?post=2174543#comment21511379

One of the two fixes will work.

**Comment by ButchDeLoria** [comment]
It's not just laptops with hybrid graphics running into this, this also affects users running BYOND with Windows virtual machines and possibly through WINE. Unfortunately neither of those workarounds would work in this case.

**Comment by Archie700** [comment]
Even my laptop with ONE Intel card has run into white and dark screens.

**Comment by AnturK** [comment]
Please post specific hardware/os you have if you're experiencing this.

**Comment by Archie700** [comment]
test
screenshot 2016-11-28 02 48 09

**Comment by HankMoodyExperience** [comment]
Works on MY Machine™

**Comment by vista-** [comment]
reeeeeeeeeee revert it now, i cannot spaceman

Intel 3rd gen i5 (Ivy Bridge) iGPU + i915 drivers + Wine 1.8.3
same HW with both KVM and VirtualBox running W7
